bin/run-integration

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 if [[ "$RUN_ARGS" == "--nightly" ]];
 then
-    MOCHA_FILE=./test-results-integration.xml npm run test-integration -- -R mocha-junit-reporter -t $CIRCLE_NODE_TOTAL -i $CIRCLE_NODE_INDEX
+	npm run test-integration:ci
```

Comment:
As a first step have we tried using the `files` option for server/integration so these at least don't run on the other nodes?

https://circleci.com/docs/1.0/parallel-manual-setup/#using-the-files-configuration-modifier